airbyte-integrations/connectors/source-mysql/src/test/java/io/airbyte/integrations/source/mysql/MySqlSourceTests.java

```diff
@@ -197,4 +198,13 @@ CREATE VIEW test_view_null_cursor(id) as
 
   }
 
+
+  @Test
+  void testParseJdbcParameters() {
```

Comment:
Maybe in addition to this, we can add a new test that given an existing source config containing a jdbc url param looking like `sessionVariables=MAX_EXECUTION_TIME=123456666`, that we are successfully able to connect to the database? So, it will be a little more e2e

As someone reading the unit tests, I'm not sure how `parseJdbcParameters` fits into the larger picture

Reply:
Added